README.md

```diff
@@ -1,38 +1,97 @@
+<path opacity="0.1" d="M9.23428 6.57916L7.95171 9.15294L7.52418 10.1539L6.52662 11.4407C6.52662 11.4407 5.81407 12.7276 5.67156 12.8706C5.52905 13.0136 4.53149 13.2996 4.53149 13.2996L5.95658 14.7295" fill="black"/>
+<path opacity="0.1" d="M3.53402 16.4454L3.24897 18.3043L3.53402 19.4482V21.45L2.67896 25.1677L3.53402 26.1686L4.53156 27.4555" fill="black"/>
+<path opacity="0.1" d="M5.67163 17.3035L6.52669 17.7324L7.09671 19.5913L8.23682 19.3053L8.80683 20.3062" fill="black"/>
+<path opacity="0.1" d="M21.3475 35.8918L23.6277 35.6058L24.6252 34.6049H26.0503" fill="black"/>
+<path opacity="0.1" d="M7.52417 32.889L8.37923 33.8899L9.80431 34.3189L10.8019 34.6049L11.0869 36.1777" fill="black"/>
+<path opacity="0.1" d="M24.3402 28.0274L25.3378 27.3125L25.7654 26.1685L26.6204 26.0256L27.0479 24.5957" fill="black"/>
+</svg>
 
-First, run the development server:
+</div>
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
+##_Shoppingify Client_
+
+---
+
+###Overview
+![Imgur](https://i.imgur.com/GounMss.png)
+
+Shoppingify allows you to take your shopping list wherever you go. With Shoppingify, you can easily create personalized shopping lists for any occasion, from weekly grocery shopping to birthday gifts. You can also organize your lists by categories and add useful notes, such as product sizes and preferences. Shoppingify can be accessed on both mobile devices and computers.
+
+---
+
+###Technologies
+:computer: The technologies chosen for this project are:
+
+- <img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" alt="descrição da imagem" width="13"> Typescript
+- <img src="https://cdn-icons-png.flaticon.com/512/875/875209.png" alt="descrição da imagem" width="13"> React
+- :zap: ChakraUI
+- <img src="https://cdn.worldvectorlogo.com/logos/next-js.svg" alt="descrição da imagem" width="13"> Next
+
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+###How to contribute
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+:memo: If you want to [contribute](https://github.com/joao-pedrozo/shoppingify-client/wiki/How-to-contribute%3F "Contribute") to the project, follow these step-by-step instructions to open a pull request.  
+:question: If you have any questions or issues, you can contact us on [Discord](https://t.co/QWAGh2nd9j).
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+---
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+###How to set up the project
+This is a Next.js project bootstrapped with create-next-app.
```

Comment:
`This is a Next.js project bootstrapped with create-next-app.`
I think we can remove this text in order to make it look cleaner